docs/en/compute/table.md

```diff
@@ -0,0 +1,224 @@
+services to determine which, and how much data to read when executing queries.
+They store information about different database catalogs, tables, storage formats, data
+location, and more.
+{{feature_name}} is designed to make it simple and straightforward to retrieve and serve structured table
+metadata to these SQL engines.
+
+
+## Architecture
+
+{{feature_name}} is designed in a way very similar to the normal Alluxio filesystem.
+The service itself is not responsible for retaining all data, but is rather a caching service for
+metadata that originates in another location (i.e. MySQL, Hive).
+These are called **UDBs**.
+UDBs are responsible for the management and storage of the metadata.
+Currently, Hive is the only supported UDB.
+{{feature_name}} caches and makes the metadata available universally through the Alluxio filesystem
+namespace.
+
+```
+Query Engine     Metadata service            Under meta service
++--------+       +-------------------+       +----------------+
+| Presto | <---> | Alluxio {{feature_name}} | <---> | Hive Metastore |
++--------+       +-------------------+       +----------------+
+```
+
+Users that have tables which span multiple storage services (i.e. AWS S3, HDFS, GCS) - would
+typically need to configure their SQL engines to connect to each one of these services individually
+in order to make requests.
+Using {{feature_name}}, all the user needs to do is configure a single Alluxio client, and data any
+supported under storage systems locations can be served and read through Alluxio.
+
+## Using {{feature_name}}
+
+### Alluxio configuration
+
+To enable {{feature_name}}, add the following line to your `alluxio-site.properties`
+
+```properties
+alluxio.table.enabled=true
+```
+
+By default, mounted databases and tables will exist underneath `/catalog` directory in Alluxio.
+Configure the root directory for structured data {{feature_name}} by configuring
+
+```properties
+alluxio.table.catalog.path=</desired/alluxio/path>
+```
+
+### Attaching Databases
+
+In order for Alluxio to serve information about structured table data, Alluxio must be informed
+about where databases are located.
+Use the Table CLI to perform any actions regarding attaching, browsing, or detaching
+databases from Alluxio.
+
+```console
+$ ${ALLUXIO_HOME}/bin/alluxio table
+Usage: alluxio table [generic options]
+         [attachdb <dbName> <dbType> [-o|--option <key=value>]]
+         [detachdb <db name>]
+         [ls [<database_name> [<table_name>]]]
+```
+
+To attach a database use the `attachdb` command. Currently, only `hive` is supported as the
+`dbType`.
+This command maps the hive database `default` into a database in Alluxio called `alluxio_db` from
+the metastore located at `thrift://metastore_host:9083`
+
+```console
+$ ${ALLUXIO_HOME}/bin/alluxio table attachdb alluxio_db hive \
+    --option udb-hive.database-name=default \
+    --option udb-hive.metastore.uris=thrift://metastore_host:9083
```

Comment:
i think the option has been changed to `hive.hive.metastore.uris` ?

Reply:
updated